test/sd_notify_test.rb

```diff
@@ -0,0 +1,58 @@
+require 'test_helper'
+require 'proxy/sd_notify'
+
+class SdNotifyTest < Test::Unit::TestCase
+  def test_active_with_notify_socket
+    with_notify_socket('/run/systemd/notify') do
```

Comment:
Ahh, yes, Proxy::SdNotify.new.active? only checks that the environment variable isn't nil. The change made it clear(er) though, thanks.